modules/swagger-codegen/src/main/java/com/wordnik/swagger/codegen/DefaultCodegen.java

```diff
@@ -621,6 +621,8 @@ public CodegenOperation fromOperation(String path, String httpMethod, Operation
         count += 1;
         if (count < operation.getConsumes().size())
           mediaType.put("hasMore", "true");
+        else
+          mediaType.put("hasMore", null);
```

Comment:
this change is to remove redundant trailing comma, e.g. change the following generated code

```java
    String[] contentTypes = {
      "application/json","application/xml",
    };
```

to 

```java
    String[] contentTypes = {
      "application/json","application/xml"
    };
```